sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -151,6 +153,7 @@ def _create_handler(
                 amqp_transport=self._amqp_transport,
             ),
             msg_timeout=self._timeout * self._amqp_transport.TIMEOUT_FACTOR,
+            use_tls=self._use_tls,
```

Comment:
can use self._client._config.use_tls instead of self._use_tls everywhere and remove above so that it's easier to make a change to just the one place if needed.

Reply:
since we're only making this change to pyamqp, would we just want to pass use_tls=config.use_tls in pyamqp_transport.py? or nah?